src/components/Box.jsx

```diff
@@ -1,5 +1,11 @@
 const Box = (props) => {
 
+    if(props.detail){
```

Comment:
This would make our `Box` component less flexible. Instead of passing the detail prop, you should allow a width to be passed. Just like we did in the previous exercise.